### PR TITLE
Search functionality

### PR DESCRIPTION
**Pull request by bryceholcomb**

### CODE REVIEW

File: app/views/application/_home_page_search_bar.html.erb [hunk 1]
```diff
@@ -1,60 +1,11 @@
...
+  <div class="form-inline">
+    <%= bootstrap_form_tag url: "/listings", method: "get" do |f| %>
+      <%= f.text_field :city, hide_label: true, placeholder: "Where to?", class: "connector" %>
+      <%= f.text_field :check_in, hide_label: true, placeholder: "Check In", class: "from_date connector", contenteditable: false %>
+      <%= f.text_field :check_out, hide_label: true, placeholder: "Check Out", class: "to_date connector", contenteditable: false %>
+      <%= f.select :max_guests, options_for_select(guests_helper), hide_label: true, class: "connector", placeholder: "Guests" %>
+      <%= f.submit "Search", class: "btn btn-primary connector" %>
+    <% end %>
+  </div>
+</div>
```
Comment (kpearson) [inline app/views/application/_home_page_search_bar.html.erb]: Nice work with the bootstrap form

File: app/views/home/index.html.erb [hunk 2]
```diff
@@ -4,9 +4,11 @@
   <div class="center-block">
     <%= link_to  listings_path do %>
     <h1 class="welcome">Destinations</h1>
+    <% if current_user %>
     <button type="button" class="btn btn-lg btn-default create-listing">
-      <div class="active"><a href="#" data-toggle="modal" data-target="#new_listing-modal" id="new_listing">Create A Listing</a></div>
+      <div class="active"><%= link_to "Create A Listing", new_user_listing_path(current_user.slug), id: "new_listing" %></div>
     </button>
```
Comment (kpearson) [inline app/views/home/index.html.erb]: Look much better

**Comment by cluhring** [comment]
thanks for explaining the unauthenticated_guest_spec.rb tests.  Nice partials!

**Comment by cluhring** [comment]
:tired_face:

**Pull request merged**